### ISSUE TITLE
Running Class-Agnostic Detector (CAD) with LOST on custom unlabeled dataset

### ISSUE THREAD

**Issue by CS-Mohammed-K** [valeoai/LOST#14]
I am trying to run LOST + CAD on a set of unlabeled dataset I have. Is it possible to do that without any annotation? 

Correct me if I am wrong, I need to get the LOST pseudo-boxes by running main_lost.py which should produce a .pkl file and results.text (which will be "nan" because I have no annotations, correct?)

After that I follow the instructions under "Training a Class-Agnostic Detector (CAD) with LOST pseudo-annotations". From what I read in the paper that this is fully unsupervised, but after trying to run the code and edit it to fit my dataset, I can see that GT annotation is needed? 

Formatted my images into COCO style, and created an instance.json file for it, but in the annotation part there is only image ID and no segmentation.

TLDR;

I have unlabeled dataset, with no annotations, I want to run LOST+CAD fully unsupervised, is it possible? 

In the image below is the message I get when I try to run "python main_lost.py --dataset DODO --set train --arch vit_small", DODO is the name of my dataset, I basically am using the COCO20k classes in the code but renamed and just changed the root_path.

![image](https://github.com/valeoai/LOST/assets/47324277/f3d0c652-f691-48b1-ab81-cbbbee586f8f)

**Comment by CS-Mohammed-K** [comment]
Running main_lost.py to obtain the pesudo-boxes return a 1kb .pkl file, which doesn't contain any pesudo-box when trying to later convert it to detectron2 format.

**Comment by CS-Mohammed-K** [comment]
Is it possible to add a script that takes custom unlabeled data and apply LOST + CAD on it? Or any guidance would be helpful.
